doc/Language/objects.pod6

```diff
@@ -263,13 +260,13 @@ Method names can be resolved at runtime with the C<.""> operator.
 =head2 self
 
 Inside a method, the term C<self> is available, which is bound to the
-invocant, i.e.  the object that the method was called on. C<self> can be
+invocant, the object that the method was called on. C<self> can be
```

Comment:
How about:
Inside a method, the term C<self> is available, which is bound to the
invocant ( the object that the method was called on). 

[I hate to think that I'm not getting it...]